nano/node/blockprocessor.hpp

```diff
@@ -86,7 +86,7 @@ class block_processor final
 	};
 
 public:
-	block_processor (nano::node &, nano::write_database_queue &);
+	block_processor (nano::node &, nano::store::write_queue &);
```

Comment:
Given that write_queue is now part of store object, is it necessary to separately pass it?

Reply:
Good point, not sure why they were being passed in. Fixed.